crates/common/tedge_utils/Cargo.toml

```diff
@@ -15,7 +15,8 @@ logging = ["tracing", "tracing-subscriber"]
 [dependencies]
 anyhow = "1.0"
 futures = "0.3"
-nix = "0.23.1"
+inotify_crate = { version = "0.10", package = "inotify" }
```

Comment:
Is the used crate https://crates.io/crates/inotify? Why not the one below?
```
inotify = "0.10"
```

Reply:
That's right, the [nix crate also has inotify](https://docs.rs/nix/latest/nix/sys/inotify/index.html) but the API is slightly different.

However, I notice now that as of nix 0.24 the inotify is a feature, meaning that the namespace will not overlap anymore. I will update nix to 0.24 and use inotify 0.10 as @rina23q  shows above